client/components/FormattingBar/controlComponents/ControlsTable.tsx

```diff
@@ -32,6 +31,7 @@ const buttonCommands = [
 	{ key: 'table-split-cell', title: 'Split cells', icon: 'split-columns' },
 	{ key: 'toggle-header-cell', title: 'Toggle header cells', icon: 'header' },
 	{ key: 'table-delete', title: 'Remove table', icon: 'trash' },
+	{ key: 'table-toggle-label', title: 'Toggle label', icon: 'tag' },
```

Comment:
Use of the `tag` icon is tentative here until we can figure out the best symbol/shortcut to use for references.